distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Cloud_Caching_Settings.enso

```diff
@@ -0,0 +1,32 @@
+import project.Data.Time.Duration.Duration
+import project.Enso_Cloud.Internal.Existing_Enso_Asset.Asset_Cache
+import project.Nothing.Nothing
+
+polyglot java import org.enso.base.enso_cloud.CacheSettings
+
+## UNSTABLE
+   ADVANCED
```

Comment:
lets put these as PRIVATE for now as don't think we want them to show in the component browser yet.

Reply:
Ok